developer-extension/src/panel/hooks/useEvents/eventFilters.ts

```diff
@@ -45,17 +46,30 @@ export function applyEventFilters(filters: EventFilters, events: SdkEvent[], fac
 
 function filterExcludedFacets(
   events: SdkEvent[],
-  excludedFacetValues: ExcludedFacetValues,
+  facetValuesFilter: FacetValuesFilter,
   facetRegistry: FacetRegistry
 ): SdkEvent[] {
-  return events.filter(
-    (event) =>
-      !Object.entries(excludedFacetValues).some(([facetPath, excludedValues]) =>
-        (excludedValues as Array<FieldMultiValue | undefined>).includes(
+  if (facetValuesFilter.type === 'exclude') {
+    const excludedFacetValues = facetValuesFilter.facetValues
+    return events.filter(
+      (event) =>
+        !Object.entries(excludedFacetValues).some(([facetPath, excludedValues]) =>
+          (excludedValues as Array<FieldMultiValue | undefined>).includes(
+            facetRegistry.getFieldValueForEvent(event, facetPath)
+          )
+        )
+    )
+  } else if (facetValuesFilter.type === 'include') {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is there a case where the filter is 'include'?

Reply:
When it is in `only` mode, we use the `include` mode of the filter.